test/unit/uk/gov/hmrc/tariffclassificationfrontend/controllers/CaseControllerSpec.scala

```diff
@@ -51,9 +51,10 @@ class CaseControllerSpec extends WordSpec with Matchers with GuiceOneAppPerSuite
   private val eventService = mock[EventsService]
   private val operator = mock[Operator]
   private val event = mock[Event]
+  private val decisionForm = new DecisionForm(new CommodityCodeConstraints(new CommodityCodeService))
```

Comment:
And again.  Or even mock the DecisionForm as it doesn't look like it's being used in the test

Reply:
I've mocked the CommodityCodeService